corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_property.html

```diff
@@ -78,6 +81,14 @@ <h4 class="modal-title">
                       </select>
                     </div>
                   </div>
+                  <div class="form-group" data-bind="visible: appearanceFinal() == 'fixture'">
+                    <label class="control-label col-xs-12 col-sm-3">
+                      {% trans "Is Multi Select" %}
```

Comment:
nit: Suggested rewording
```suggestion
                      {% trans "Allow multiple selections" %}
```